activity.js

```diff
@@ -14,12 +14,13 @@ class Activity {
     }
 
     markComplete() {
-        if (this.seconds < 0) {
-            this.completed = true;
-        }
+        this.completed = true;
```

Comment:
We should come back to this and consider if we are updating the data model vs the DOM, and possibly refactor a bit.